src/components/IconSetPreviewFooter.tsx

```diff
@@ -224,6 +228,16 @@ const IconSetPreviewFooter = ({
                 <Icon icon="download" size={20} />
               </Button>
             </Tooltip>
+            <Tooltip message="Download icon types for Typescript">
+              <Button
+                disabled={selectedIcons.length === 0}
+                variant="icon"
+                onClick={handleDownloadIconTypes}
+              >
+                <Icon icon="filetype-tsx" size={20} />
+                <Icon icon="download" size={20} />
+              </Button>
+            </Tooltip>
```

Comment:
I don't want to enable this feature for now. You can remove this button.

Reply:
Is there any roadmap for this project?

Reply:
We use svgps.app on production and this feature would be so helpful to manage icons with typescript 👉👈

Reply:
I have things planned, but there is no ROADMAP released yet.